ts/Stock/Indicators/MultipleLinesComposition.ts

```diff
@@ -167,6 +198,46 @@ namespace MultipleLinesComposition {
         return IndicatorClass as (T&typeof Composition);
     }
 
+
+    /**
+     * function to create the path based on the points
+     * @param this Indicator
+     * @param points Points on which the path should be created
+     */
+    function getGraphPath(this: Composition, points: Array<LinePoint>): SVGPath {
+        let indicator = this,
+            path: SVGPath = [],
+            spanA: SVGPath,
+            spanAarr: SVGPath = [];
+
+        points = points || this.points;
+
+        // Render Span
+        if (indicator.fillGraph && indicator.nextPoints) {
+            spanA = SeriesRegistry.seriesTypes.sma.prototype.getGraphPath.call(
+                indicator,
+                // Reverse points, so Senkou Span A will start from the end:
+                indicator.nextPoints
+            );
+
+            if (spanA && spanA.length) {
+                spanA[0][0] = 'L';
+
+                path = SeriesRegistry.seriesTypes.sma.prototype.getGraphPath.call(indicator, points);
+
+                spanAarr = spanA.slice(0, path.length);
+
+                for (let i = spanAarr.length - 1; i >= 0; i--) {
+                    path.push(spanAarr[i]);
```

Comment:
Q: Could you remind me the format of paths in indicators? Is it:
- `[ 'L', x, y, 'L', x, y, ... ]` or:
- `[ [ 'L', x, y ], [ 'L', x, y ], ... ]`?

Reply:
Second one :D

Reply:
Thanks! :)